### PR TITLE
Create utcwebdev-chattanooga.md

### LINKED ISSUE #36: UTCWebDev - Chattanooga, TN
# UTCWebDev - Chattanooga, TN
- [x] Create a branch
- [x] Add File
- [x] Create a commit
- [x] Open a PR
- [ ] Have a discussion about PR
- [ ] Merge PR

## Header Text

Plain old test text.

- with bullets
- like this

**bold text**
_italic_

### PR DESCRIPTION
**Pull request by UTCWebDev**
closes #36

### CODE REVIEW

**Comment by kha3474** [comment]
good

File: places/utcwebdev-chattanooga.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Chattanooga, TN
+
+## Fun Facts
+- it is the scenic city
```
Comment (jwsocal) [inline places/utcwebdev-chattanooga.md]: what makes it so scenic?

File: places/utcwebdev-chattanooga.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Places to Eat
+- Lupi's Pizza
+
+## Places to Go
+- Rock City
+
+## Places to Avoid
+- Soddy Daisy
```
Comment (jwsocal) [inline places/utcwebdev-chattanooga.md]: what is Soddy Daisy?

File: places/utcwebdev-chattanooga.md (hunk 1 above)
Comment (UTCWebDev) [inline places/utcwebdev-chattanooga.md]: Mountains and rivers, mostly.

File: places/utcwebdev-chattanooga.md (hunk 2 above)
Comment (UTCWebDev) [inline places/utcwebdev-chattanooga.md]: SD is a notoriously backward area in the county; it's basically the butt of jokes because of its name.

**Comment by jwsocal** [comment]
sounds good to me! 🚢

**Pull request merged**